btcec/signature_test.go

```diff
@@ -35,320 +35,63 @@ func decodeHex(hexStr string) []byte {
 	return b
 }
 
-var signatureTests = []signatureTest{
```

Comment:
Did you keep the struct `signatureTest` on purpose? I think it isn't used anymore.